### ISSUE TITLE
Kobo DRM Removal Error

### ISSUE THREAD

**Issue by Tamarissa** [apprenticeharper/DeDRM_tools#946]
I've never had a problem getting my books off of my device, but now it no longer works. The 'text' of the books with DRM looks like this:
�	��OG[D��T� Mcy9�BG����x����������X��$*�=��Q.n���P����]�Za.3m�+��i�H�������M7p�3lm�Y��rR�(/��g 

When I tried to get a log as per the provided instructions, the debug command is not recognized.

which tool was used - Obok deDRM 6.5.4 Kobo
which OS you are using - Mojave  10.14.6
where the ebook file came from (device or application) - Kobo eReader
any error messages - none
a log of the DRM-removal attempt (see the FAQs on how to get a log). 
 calibre-debug -g
-bash: calibre-debug: command not found
<img width="408" alt="Screen Shot 2019-12-01 at 2 28 17 AM" src="https://user-images.githubusercontent.com/58387230/69912422-5eaca500-13e6-11ea-908a-dc3696248eca.png">

**Comment by ElleKayEm** [comment]
Here's how to get a log:

https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md#i-cannot-solve-my-problem-with-the-dedrm-plugin-and-now-i-need-to-post-a-log-how-do-i-do-that

**Comment by Tamarissa** [comment]
Thanks, that was more useful than the part on that page that said to us the terminal to get the log...

[tmpGF28Cv.txt](https://github.com/apprenticeharper/DeDRM_tools/files/3908628/tmpGF28Cv.txt)

**Comment by ElleKayEm** [comment]
Hmm, Obok seems to think it did successfully decrypt the book.  I know it normally won't even add a book to calibre if it can't remove the DRM.  How are you trying to view the book?

**Comment by Tamarissa** [comment]
I just click on the 'epub' format to open it up with the ebook viewer.
I guess technically it adds a book, but there's no text, no cover... just pages and pages of those random letters/numbers/symbols. It did produce a ToC though; seems weird it can get that right but not the text.

**Comment by ElleKayEm** [comment]
Calibre's viewer shouldn't open anything with DRM, but obviously something went wrong.  They recently made major changes to the viewer, but this looks like more than a display problem.  Maybe look at it in a different epub reader just to be sure.

Don't really know what to tell you to try next.  I guess I'd try reinstalling Obok first.  Then maybe try Kobo Desktop, instead of your Kobo device.  If that doesn't work, you could always download the Adobe DRM epubs from the website via Adobe Digital Editions 2.0.1 and then use the DeDRM plugin.

**Comment by ElleKayEm** [comment]
Another thought -- do the DRMed books look fine on your Kobo to begin with?

**Comment by Tamarissa** [comment]
Well, I tried opening it in Books, and it's definitely not a dislpay issue unfortunately.
<img width="443" alt="Screen Shot 2019-12-01 at 7 15 22 PM" src="https://user-images.githubusercontent.com/58387230/69925804-12597780-146f-11ea-9a61-9de6757818d0.png">

I'd already reinstalled Obok before I started this... so that didn't work either.

I'm working on trying it from the Kobo Desktop app, but I never use it, so it's taking a long time to go through the sync process. If that doesn't work, I'll download Adobe and try it that way.

The book does look fine on my reader...
